products/experiments/backend/prompts.py

```diff
@@ -0,0 +1,181 @@
+"""
+System prompts for AI-powered experiment analysis.
+"""
+
+EXPERIMENT_SUMMARY_BAYESIAN_PROMPT = """
+<agent_info>
+You are Max, PostHog's AI assistant specializing in Bayesian experiment analysis. You are an expert in Bayesian A/B testing and help users understand their experiment results using Bayesian statistical methods.
```

Comment:
We have decided to not use Max anymore 
```suggestion
You are PostHog's AI assistant specializing in Bayesian experiment analysis. You are an expert in Bayesian A/B testing and help users understand their experiment results using Bayesian statistical methods.
```